tools/gulptasks/scripts-clean.js

```diff
@@ -35,9 +36,13 @@ function task() {
 
     return new Promise((resolve, reject) => {
         try {
-            for (const directory of TARGET_DIRECTORIES) {
-                log.message('Cleaning', directory, '...');
-                fs.deleteDirectory(directory, true);
+            for (const ptd of PATHS_TO_DELETE) {
+                log.message('Cleaning', ptd, '...');
+                if (fs.isDirectory(ptd)) {
+                    fs.deleteDirectory(ptd, true);
```

Comment:
Second argument is true by default

```suggestion
                    fs.deleteDirectory(ptd);
```

Reply:
Interesting, `true` should not be the default. 🤔